pom.xml

```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.baidu.hugegraph</groupId>
+    <artifactId>hugegraph-commons</artifactId>
+    <version>1.8.10</version>
+    <packaging>pom</packaging>
+
+    <name>hugegraph-commons</name>
+    <url>https://github.com/hugegraph/hugegraph-commons</url>
+    <description>
+        hugegraph-commons is a common module for HugeGraph-Common and HugeGraph-Rpc with their peripheral components.
+        It includes rpc frame, locks, configurations, events, iterators, rest and some
+        numeric or collection util classes to simplify the development of HugeGraph and its components.
+    </description>
+
+    <parent>
+        <groupId>org.sonatype.oss</groupId>
+        <artifactId>oss-parent</artifactId>
+        <version>7</version>
+    </parent>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <url>https://github.com/hugegraph/hugegraph-commons</url>
+        <connection>https://github.com/hugegraph/hugegraph-commons</connection>
+        <developerConnection>https://github.com/hugegraph/hugegraph-commons</developerConnection>
+    </scm>
+
+    <developers>
+        <developer>
+            <name>lizhangmei</name>
+            <email>javaloveme@gmail.com</email>
+        </developer>
+        <developer>
+            <name>zhoney</name>
+            <email>zhangyi89817@126.com</email>
+        </developer>
+        <developer>
+            <name>liningrui</name>
+            <email>liningrui@vip.qq.com</email>
+        </developer>
```

Comment:
can add more developers

Reply:
it's optional & maybe do it in a separate pr (if need)